src/_settings.scss

```diff
@@ -474,3 +475,13 @@ $size-topbar-user-toggle-icon: 24px !default;
 $w-topbar-user-nav: 90% !default;
 $maw-topbar-user-nav: 350px !default;
 $bgc-topbar-user-cta: $c-accent !default;
+
+// Thumbnail
+$bgc-thumbnail: $c-white !default;
+$bd-thumbnail: 1px solid $c-gray-lightest !default;
+$p-thumbnail: ceil($p-base / 6) !default;
```

Comment:
It's a possibility, but it seems a little strange to me to use two different vars for the same value. In this case: `$p-thumbnail: $p-v-xsmall $p-h-xsmall;` Also, this values are different: 3px and 6px respectly. We want the same padding for horitzontal and vertical. 
Maybe there are a third way: create new vars: `$p-base-small: ceil($p-base / 2) !default;` `$p-base-xsmall: ceil($p-base / 3) !default;` `$p-base-xxsmall: ceil($p-base / 4) !default;`